### PR TITLE
Warp command auto-complete

### PR DESCRIPTION
**Pull request by viciscat**
I will not take inquiries for my poor coding decisions.

reasons:
1. I'm tired and I want to sleep and I have school tomorrow
2. Reason 2
3. bob

### CODE REVIEW

File: src/main/resources/skyblocker.mixins.json [hunk 1]
```diff
@@ -47,7 +47,8 @@
     "accessor.ScreenAccessor",
     "accessor.SkullBlockEntityAccessor",
     "accessor.WorldRendererAccessor",
-    "discordipc.ConnectionMixin"
+    "discordipc.ConnectionMixin",
+    "CommandTreeS2CPacketMixin"
```
Comment (AzureAaron) [inline src/main/resources/skyblocker.mixins.json:51]: The list is supposed to be alphabetized :<

File: src/main/java/de/hysky/skyblocker/mixin/CommandTreeS2CPacketMixin.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import net.minecraft.command.CommandSource;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+
+@Mixin(targets = "net.minecraft.network.packet.s2c.play.CommandTreeS2CPacket$CommandTree")
+public class CommandTreeS2CPacketMixin {
+
+    @ModifyExpressionValue(method = "getNode", at = @At(ordinal = 1, value = "INVOKE", target = "Lnet/minecraft/network/packet/s2c/play/CommandTreeS2CPacket$CommandTree;getNode(I)Lcom/mojang/brigadier/tree/CommandNode;"))
+    public CommandNode<? extends CommandSource> thing(CommandNode<CommandSource> original) {
+        if (WarpAutocomplete.COMMAND_THING != null && original instanceof LiteralCommandNode<?> literalCommandNode && literalCommandNode.getLiteral().equals("warp") && (Utils.isOnHypixel())) {
```
Comment (AzureAaron) [inline src/main/java/de/hysky/skyblocker/mixin/CommandTreeS2CPacketMixin.java:17]: `Utils.isOnHypixel` doesn't need to be wrapped in brackets

File: src/main/java/de/hysky/skyblocker/skyblock/WarpAutocomplete.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * the mixin {@link de.hysky.skyblocker.mixin.CommandTreeS2CPacketMixin}
+ */
+public class WarpAutocomplete {
+
```
Comment (AzureAaron) [inline src/main/java/de/hysky/skyblocker/skyblock/WarpAutocomplete.java:28]: could remove this whitespace line

**Review by AzureAaron — CHANGES_REQUESTED** [review]

**Review by kevinthegreat1 — APPROVED** [review]
Works well

**Pull request merged**